<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSensorForm</class>
 <widget class="QWidget" name="WidgetSensorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBox">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="color_widgets::ColorSelector" name="ColorSelector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="bShowGraph">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>color_widgets::ColorPreview</class>
   <extends>QWidget</extends>
   <header>color_preview.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorSelector</class>
   <extends>color_widgets::ColorPreview</extends>
   <header>color_selector.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
